README.md

```diff
@@ -1,83 +1,85 @@
 Riak Dev Cluster for OS X
 =========================
 
-Easily run a 5-node [Riak](http://wiki.basho.com/Riak.html) cluster on
-OS X.
+*NOTE: This project currently installs Riak 2.0.5*
 
+Easily run a 5-node [Riak](http://www.basho.com/riak) cluster on OS X.
+
+The Basho documentation enumerates the [Five Minute Install](http://docs.basho.com/riak/latest/quickstart/). While a straightforward, and useful, exercise to undergo the tutorial is based upon a source build. As a result, in some environments, it can be a time-intensive process.
```

Comment:
Yes, I was being overly cautious.  Thanks for the suggestion on the rewrite will update and commit.